Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -196,6 +196,11 @@ class AppSettingsTableViewController: SettingsTableViewController, FeatureFlagga
                      .init(title: .init(string: .localized(.customization)), children: customization),
                      .init(title: .init(string: .SettingsGeneralSectionTitle), children: generalSettings)]
         
+        // Ecosia: Disable search settings for Bing search experiment when we show the Bing SERP
+        if BingDistributionExperiment.shouldShowBingSERP {
+            settings.removeAll(where: { $0.title?.string == .localized(.search) } )
+        }
```

Comment:
Isn't this only `searchSettings`? Maybe it would be more readable if we only appended them `if !BingDistributionExperiment.shouldShowBingSERP` instead of afterwards. What do you think?

Reply:
I second @lucaschifino's thought. I know the code works but it seems risky to rely on string match like `$0.title?.string == .localized(.search)`  to filter the array of settings afterwards. I'd also favour a more explicit logic of not adding it instead.

Reply:
I'm happy with either 👍 
I readded it like that from the other code as that looked easier to remove.